teamworkApp/lib/importData.py

```diff
@@ -1,26 +1,260 @@
+			answer_to_db.append((
+				response,
+				student_dict[student][0],
+				now,
+				primary_keys[count],
+				question_count
+			))
+			question_count += 1
+		count += 1
+	return answer_to_db
 
+def list_style_data(student_dict):
+	"""Given a dictionary of student data, calculate and import style data
+	for each student.
+	Uses find_style to calculate styles."""
+	style_to_db = []
+	# fetch the names of the students from the table
+	conn = sqlite3.connect(DB)
+	c = conn.cursor()
+	# get the primary keys for the students
+	c.execute('SELECT id FROM students;')
+	temp_keys = c.fetchall()
+	conn.commit()
+	conn.close()
+
+	primary_keys = [temp_key[0] for temp_key in temp_keys]
+
+	count = 0
+	for student in student_dict.keys():
+		now               = datetime.now().isoformat()
+		expanded_responses = student_dict[student][1]
+		communicator, collaborator, challenger, contributor = find_style(expanded_responses)
+		# TODO: The commented out line below should be what we use in the
+			# future; however, evaluateAnswers has not been updated for the
+			# new schema (as of the time of writing this 10-28-2017) so we
+			# cannot use it yet
+		# (communicator, collaborator, challenger, contributor) = find_scores(primary_keys[count])
+		created_at   = student_dict[student][0]
+		student_id   = primary_keys[count]
+		style_to_db.append((
+				student_id,
+				communicator,
+				collaborator,
+				challenger,
+				contributor,
+				created_at,
+				now
+			))
+		count += 1
+
+	return style_to_db
+
+def execute_insert(csv_filename):
+	student_dict   = process_answer_data(csv_filename)
+	student_to_db = list_student_data(student_dict)
+	print("Inserting .csv information into 'students'...")
+	conn = sqlite3.connect(DB)
+	c = conn.cursor()
+	# insert student data from .csv into students
+	c.executemany('INSERT INTO students (name, created_at, updated_at, username) VALUES (?, ?, ?, ?);', student_to_db)
 	conn.commit()
 	conn.close()
+	print("Success.")
+	# answer and style depend on information being stored in students so we
+		# can only call list_answer_data and list_student_data after
+		# list_student_data's information has been inserted
+	answer_to_db  = list_answer_data(student_dict)
+	style_to_db   = list_style_data(student_dict)
+	print("Inserting data into 'answers' and 'styles'...")
+
+	conn = sqlite3.connect(DB)
+	c = conn.cursor()
+	# insert answer data for each question for the student
+	c.executemany('INSERT INTO answers (value, created_at, updated_at, student_id, question) VALUES (?, ?, ?, ?, ?);', answer_to_db)
+	# insert style data for each student
+	c.executemany('INSERT INTO styles (student_id, communicator, collaborator, challenger, contributor, created_at, updated_at) VALUES (?, ?, ?, ?, ?, ?, ?);', style_to_db)
+	conn.commit()
+	conn.close()
+
+	print("Success.")
+
+def main():
+	args = sys.argv
+	# TODO: remove this and other print statements when moving to Ruby
+	print("Inserting data)
+	execute_insert(args[1])
```

Comment:
Do we want some checks on args[1] to make sure that it is a valid input?

Reply:
Definitely will get on that thank u!